_config.yml

```diff
@@ -12,3 +12,8 @@ authors:
     display_name: Matthew
   zarino:
     display_name: Zarino
+
+# https://github.com/RobertDeRose/jekyll-livereload/
+# Enable livereload by default when running `jekyll serve` locally.
+# (Doesn't do anything on Github Pages.)
+livereload: true
```

Comment:
jekyll's incremental is sadly also rubbish and buggy. To work around this, we need to add https://github.com/dracos/web-design-standards-docs/blob/cea68ab020748776030bec991386f4a8d9ac3e60/_plugins/incremental_regeneration_fixer.rb under `_plugins` and add to _config.yml:

```
incremental_regeneration_fixer:
  interdependent_files:
    - "{_posts,blog}/*"
```
I know.